src/libsystemd/sd-bus/busctl-introspect.c

```diff
@@ -639,6 +648,21 @@ static int parse_xml_node(Context *context, const char *prefix, unsigned n_depth
 
                         break;
 
+                case STATE_SIGNAL_ARG_DIRECTION:
+
+                        if (t == XML_ATTRIBUTE_VALUE) {
+                                free(argument_direction);
+                                argument_direction = name;
+                                name = NULL;
```

Comment:
Direct copypaste of STATE_METHOD_ARG_DIRECTION. Do I change this occurence only, or all 12 of them?